### DISCUSSION TITLE
Changing images in supported helm charts to use iron bank?

### DISCUSSION THREAD

**Discussion by Ansible-man** [zarf-dev/zarf#2412] [Q&A]
I require images to be sourced from different registries and I would like to use helm charts maintained by the SW vendors. Right now I am painfully editing vendor provided helm charts at scale to pull iron bank images and then moving them over an air gap and deploying the charts there. Zarf is perfect but I do not see a way to point it at a chart URL AND have it replace the images in the chart to point at iron bank images or some other source registry. Is this already a feature and I am not finding it, should I put in a feature request, or is there a better way?

**Comment by lucasrod16** [comment]
@Ansible-man you can use the `components.images` field to specify the iron bank images you want to use for your upstream charts in your `zarf.yaml`. When you `create` the package, Zarf will pull the iron bank images into your final package tarball. When you `deploy` your package, Zarf will push those images into the registry, and the Zarf agent will mutate the image reference in all of your pod specs to pull those images from the Zarf registry.

https://docs.zarf.dev/ref/components/#container-images
